andela/bp-esa-frontend
# #162724976 Change color of active view type icon

#### What does this PR do?
Changes color of active view type icon.

#### Description of Task to be completed?
- [x] Add an active color difference for the current view type icon compared to one not selected which is lighter.

#### How should this be manually tested?
- Checkout to the ch-active-icon-166179450 branch.
- Launch the applicaiton
- Select the card view icon.
- The current selected type icon should be darker than the one not selected.

#### What are the relevant pivotal tracker stories?
[#162724976](https://www.pivotaltracker.com/story/show/166179450)

#### Screenshots (if appropriate)
##### List View 
<img width="1432" alt="Screen Shot 2019-05-22 at 20 54 57" src="https://user-images.githubusercontent.com/36948493/58227145-52a6a300-7d32-11e9-8c11-1fad1cb2cbaa.png">
##### Card View
<img width="1427" alt="Screen Shot 2019-05-22 at 20 54 47" src="https://user-images.githubusercontent.com/36948493/58227156-5cc8a180-7d32-11e9-9ad2-4c8ec46b374b.png">
